tests/suite/test_smoke.py

```diff
@@ -77,20 +81,20 @@ def fin():
     return SmokeSetup(ingress_controller_endpoint, ingress_host)
 
 
-@pytest.mark.ingresses
 @pytest.mark.smoke
-@pytest.mark.parametrize(
-    "ingress_controller",
-    [
-        pytest.param({"extra_args": ["-enable-prometheus-metrics"]}, id="one-additional-cli-args"),
-        pytest.param(
-            {"extra_args": ["-nginx-debug", "-health-status=true", "-enable-prometheus-metrics"]},
-            id="some-additional-cli-args",
-        ),
-    ],
-    indirect=True,
-)
 class TestSmoke:
+    @pytest.mark.ingresses
```

Comment:
`ingresses` marker and `ingress_controller` fixture can and should be called at class level as ic one is common to both tests and `ingresses` marker will only cause first test to run in its current form